[15:19:58] I/launcher - Running 1 instances of WebDriver
[15:19:58] I/hosted - Using the selenium server at http://localhost:4444/wd/hub
Started
null
test_step_inputadmin
test_step_inputadmin1
test_step_click Login
test_step_click Master Inventory
test_step_click All
test_step_click 
test_step_click Submit
test_step_click 
[31mF[0m

Failures:
1) PLSQL FileDownload
  Message:
[31m    Failed: Wait timed out after 40008ms[0m
  Stack:
    TimeoutError: Wait timed out after 40008ms
        at C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2201:17
        at ManagedPromise.invokeCallback_ (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
        at TaskQueue.execute_ (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
        at TaskQueue.executeNext_ (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
        at C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
        at C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    From: Task: <anonymous wait>
        at scheduleWait (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2188:20)
        at ControlFlow.wait (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2517:12)
        at thenableWebDriverProxy.wait (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:934:29)
        at UserContext.<anonymous> (D:\iTAP\Recorded_Scenarios\PLSQL\FileDownload\spec.js:302:16)
        at C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
        at new ManagedPromise (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
        at ControlFlow.promise (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
        at schedulerExecute (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
        at TaskQueue.execute_ (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
        at TaskQueue.executeNext_ (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    From: Task: Run it("FileDownload") in control flow
        at UserContext.<anonymous> (C:\Users\es00780451\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    From asynchronous test: 
    Error
        at Suite.<anonymous> (D:\iTAP\Recorded_Scenarios\PLSQL\FileDownload\spec.js:8:21)
        at Object.<anonymous> (D:\iTAP\Recorded_Scenarios\PLSQL\FileDownload\spec.js:7:15)
        at Module._compile (node:internal/modules/cjs/loader:1103:14)
        at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
        at Module.load (node:internal/modules/cjs/loader:981:32)
        at Function.Module._load (node:internal/modules/cjs/loader:822:12)

1 spec, 1 failure
Finished in 63.298 seconds

[15:21:08] I/launcher - 0 instance(s) of WebDriver still running
[15:21:08] I/launcher - chrome #01 failed 1 test(s)
[15:21:08] I/launcher - overall: 1 failed spec(s)
[15:21:08] E/launcher - Process exited with error code 1
